webapp/src/components/admin_settings/jitsi_settings.tsx

```diff
@@ -59,235 +55,201 @@ const DEFAULT_SETTINGS: Settings = {
+                    <label className='col-sm-4'>
+                        <FormattedMessage
+                            id='jitsi.server'
+                            defaultMessage={'Server:'}
+                        />
+                    </label>
+                    <div className='col-sm-8'>
+                        <label className='radio-inline pt-0'>
+                            <input
+                                type='radio'
+                                checked={mode === JITSI_MODE}
+                                onChange={onModeSelected}
+                                value={JITSI_MODE}
+                            />
                             <FormattedMessage
-                                id='jitsi.server'
-                                defaultMessage={'Server:'}
+                                id='jitsi.input-enable-jitsi'
+                                defaultMessage='Jitsi'
                             />
                         </label>
-                        <div className='col-sm-8'>
-                            <label className='radio-inline pt-0'>
-                                <input
-                                    type='radio'
-                                    checked={this.state.mode === JITSI_MODE}
-                                    onChange={this.onModeSelected}
-                                    value={JITSI_MODE}
-                                />
-                                <FormattedMessage
-                                    id='jitsi.input-enable-jitsi'
-                                    defaultMessage='Jitsi'
-                                />
-                            </label>
-                            <label className='radio-inline pt-0'>
-                                <input
-                                    type='radio'
-                                    checked={this.state.mode === JAAS_MODE}
-                                    onChange={this.onModeSelected}
-                                    value={JAAS_MODE}
-                                />
+                        <label className='radio-inline pt-0'>
+                            <input
+                                type='radio'
+                                checked={mode === JAAS_MODE}
+                                onChange={onModeSelected}
+                                value={JAAS_MODE}
+                            />
+                            <FormattedMessage
+                                id='jitsi.input-enable-jass'
+                                defaultMessage='JasS'
+                            />
+                        </label>
+                        <div className='help-text'>
+                            <span>
                                 <FormattedMessage
-                                    id='jitsi.input-enable-jass'
-                                    defaultMessage='JasS'
+                                    id='jitsi.serever-description'
```

Comment:
```suggestion
                                    id='jitsi.server-description'
```
Fix this everywhere.